dropwizard-jersey/src/main/java/io/dropwizard/jersey/params/DurationParam.java

```diff
@@ -0,0 +1,20 @@
+package io.dropwizard.jersey.params;
+
+import io.dropwizard.util.Duration;
+
+public class DurationParam extends AbstractParam<Duration> {
```

Comment:
This isn't directly related to the PR, but we use this often and seems like something Dropwizard should include by default.